testgrid/conformance/upload_e2e.py

```diff
@@ -199,11 +208,16 @@ def parse_args(cli_args=None):
         required=False,
         action='store_true',
     )
+    parser.add_argument(
+        '--metadata',
+        help='dictionary of additional key-value pairs that will be displayed to the user.',
```

Comment:
nit: `s/will/can/`, since on testgrid you still have to configure them to be displayed, they won't necessary be displayed